modules/ckanext-ytp_main/ckanext/ytp/public/javascript/form.js

```diff
@@ -59,75 +59,97 @@ $(function() {
   });
 });
 
-ckan.module("ytp_main_input_multiple", function($) {
+/* An multi-input module that allows to add repeating text-inputs on form
+ *
+ * linkText - Text/Translation-key that should be used for add-button
+ * linkIcon - Fontawesome icon-class for i-element that is added before linkText inside add button
+ *          - Set false, if you don't want to show icon
+ * 
+ * Examples
+ *
+ *   // <input name="example" data-module="ytp_main_input_multiple" data-module-link-text="Add link" data-module-link-icon="fa-globe" />
+ *   // <input name="example" data-module="ytp_main_input_multiple" data-module-link-text="Add link" data-module-link-icon="false" />
+ *
+ */
+ckan.module("ytp_main_input_multiple", function ($) {
   return {
-    initialize: function() {
+    options: {
+      linkText: 'Add link',
+      linkIcon: 'fa-globe',
+    },
+    initialize: function () {
       var module = this;
-      var addLinkText = module._("Add link");
-      $(function() {
-        /* Create an add link for all the multiple-values child div elements. The add link clones the input container. */
-        $(".multiple-values").each(function(index, element) {
-          var listContainer = $(element);
-
-          if (listContainer.siblings('button.add-input-button').length > 0) {
-              return false;
-          }
-          // We are adding the 'add link' after the listContainer
-          var addLink = $(
-            '<button type="button" class="add-input-button suomifi-button-secondary"><i class="fa fa-globe"></i>' +
-              addLinkText +
-              "</button>"
-          );
-
-          addLink.click(function(e) {
-            var inputContainer = $(element)
-              .children("div")
-              .first();
-            // Clone the input container div which contains the input field
-            var clonedInputContainer = inputContainer.clone();
-            // Clear the input field's value and remove the id
-            clonedInputContainer
-              .find("> input")
-              .val("")
-              .removeAttr("id")
-              .removeAttr("data-module");
-
-            clonedInputContainer.addClass("removable-input-container");
-
-            // Append the cloned input container after the last element
-            listContainer.append(clonedInputContainer);
-
-            // Add the 'remove link' to the cloned input container
-            module.createRemoveLink(clonedInputContainer);
-            e.stopPropagation();
-            e.preventDefault();
-            return false;
-          });
+      var addLinkText = module._(module.options.linkText);
```

Comment:
Translation function should have the actual text in it, so move the _() call to line 77. Otherwise translation generation will not pick it up.